packages/odyssey-react/src/components/FieldGroup/FieldGroup.module.scss

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
🤔

Reply:
I think this is because this file was copied from the @okta/odyssey package.